docs/server-side-configuration.md

```diff
@@ -0,0 +1,37 @@
+
+---
+
+# Cozy MyAccounts: Server side configuration
+
+## Authorized categories list
+
+The MyAccounts categories names list won't be automatically computed from available connectors in order to keep it clear and relevant for all Cozy users with a growing number of connectors.
+
+Instead, they need to be configured on the served side, in a simple way, from a config file. This is a list of categories authorized to be used:
+
+```javascript
+{
+    authorizedCategories:
+        {
+            'insurance': 1,
```

Comment:
I'm not sure why we declare categories as object keys. I think an array of objects, containing at least a `slug` key would be easier to maintain and to enrich if needed.

By curiosity, what have you in mind when declaring `<categoryName> : 1`? (and so, what does `1` means here?)